ql/src/test/queries/clientpositive/materialized_view_create_rewrite_multi_db.q

```diff
@@ -3,6 +3,7 @@ set hive.support.concurrency=true;
 set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
 set hive.strict.checks.cartesian.product=false;
 set hive.materializedview.rewriting=true;
+set hive.materializedview.rewriting.query.text=false;
```

Comment:
Why is this disabled now? Does the test change?

Reply:
This test was written for calcite based rewrite